### PR TITLE
File sender integration tests

### PR DESCRIPTION
**Pull request by jaceko**

### CODE REVIEW

File: example-context/example-service/example-it/pom.xml [hunk 1]
```diff
@@ -170,6 +176,18 @@
             <artifactId>hamcrest-date</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.services</groupId>
+            <artifactId>file-alfresco</artifactId>
+            <version>0.35.0-SNAPSHOT</version>
```
Comment (mapingo) [inline example-context/example-service/example-it/pom.xml]: Does this need to be snapshot?

File: example-context/example-service/example-it/src/test/java/uk/gov/justice/services/example/cakeshop/it/CakeShopIT.java [hunk 2]
```diff
@@ -88,19 +86,23 @@
...
     private static final String ORDER_CAKE_MEDIA_TYPE = "application/vnd.example.order-cake+json";
     private static final String QUERY_RECIPE_MEDIA_TYPE = "application/vnd.example.recipe+json";
     private static final String QUERY_RECIPES_MEDIA_TYPE = "application/vnd.example.recipes+json";
     private static final String QUERY_ORDER_MEDIA_TYPE = "application/vnd.example.order+json";
 
-    public final static String JMS_USERNAME = "jmsuser";
+    private final static String JMS_USERNAME = "jmsuser";
 
-    public final static String JMS_PASSWORD = "jms@user123";
+    private final static String JMS_PASSWORD = "jms@user123";
```
Comment (mapingo) [inline example-context/example-service/example-it/src/test/java/uk/gov/justice/services/example/cakeshop/it/CakeShopIT.java]: looks a bit odd when everything else is static then final

File: example-context/example-service/example-it/pom.xml (hunk 1 above)
Comment (jaceko) [inline example-context/example-service/example-it/pom.xml]: fixed

File: example-context/example-service/example-it/src/test/java/uk/gov/justice/services/example/cakeshop/it/CakeShopIT.java (hunk 2 above)
Comment (jaceko) [inline example-context/example-service/example-it/src/test/java/uk/gov/justice/services/example/cakeshop/it/CakeShopIT.java]: fixed

**Pull request merged**